aggregate_spatial_window.json

```
@@ -0,0 +1,98 @@
+                            "type": "array",
+                            "items": {
+                                "description": "Any data type."
+                            }
+                        }
+                    },
+                    {
+                        "name": "context",
+                        "description": "Additional data passed by the user.",
+                        "schema": {
+                            "description": "Any data type."
+                        },
+                        "optional": true,
+                        "default": null
+                    }
+                ]
+            }
+        },
+        {
+            "name": "size",
+            "description": "Window sizes (pixels) along each dimension.\n\nFirst value correspond to Window size along X or Longitude and second to Window size along Y or Latitude.",
+            "schema": {
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "subtype": "chunk-size",
+                    "required": [
+                        "value"
+                    ],
+                    "properties": {
+                        "value": {
+                            "type": "number",
+                            "minimum": 1,
+                            "description": "The unit the values are given in pixels (`px`)"
+                        }
+                    }
+                }
+            }
+        },
+        {
+            "name": "boundary",
+            "description": "Behavior to apply if the number of values for the axes `x` and `y` is not a multiple of the corresponding `size` = [`x_size`,`y_size`] parameter. Options are:\n\n- `pad` (default): pad the data cube with `NaN` (not a number) values to fit the required window size.\n\n- `trim`: trim the data cube to fit the required window size.",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "pad",
+                    "trim"
+                ]
+            },
+            "optional": true,
+            "default": "pad"
+        },
+        {
+            "name": "context",
+            "description": "Additional data to be passed to the reducer.",
+            "schema": {
+                "description": "Any data type."
+            },
+            "optional": true,
+            "default": null
+        }
+    ],
+    "returns": {
+        "description": "A data cube with the newly computed values and the same dimensions.\n\nThe resolution will change, depending on the chosen parameters. It decreases for the dimensions which have the corresponding parameter `size` = [`x_size`,`y_size`] set to values greater than 1.\n\nThe dimension labels will be set to the center of the window. The other dimension properties (name, type and reference system) remain unchanged.",
+        "schema": {
+            "type": "object",
+            "subtype": "vector-cube"
```

It seems a couple of things have been reverted. Please go through all the comments again and check whether there are still changes to be done. I found a couple of them...

```suggestion
            "subtype": "raster-cube"
```